config/routes.rb

```diff
@@ -132,7 +132,16 @@
       patch "unsubscribe", to: "accounts#unsubscribe"
     end
     resources :login_keys, only: %i[show new create]
-    resources :jobseeker_profiles, only: %i[index show]
+    resources :jobseeker_profiles, only: %i[index show] do
+      member do
+        resources :invitations, only: :index
+        scope controller: "invitations", path: "/invite" do
+          get "", action: :start, as: :invite_to_apply
```

Comment:
I assume `get ""` will mean the path will be something like `publishers/jobseeker_profile/<id>/invite`